File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.SearchService.Models/search-training-events_item.ts


import { SearchTrainingEventLocations_Item } from './search-training-event-locations_item';
import { SearchTrainingEventKeyActivities_Item } from './search-training-event-key-activities_item';
import { ISearchTrainingEvents_Item } from './isearch-training-events_item';

export class SearchTrainingEvents_Item implements ISearchTrainingEvents_Item {
  
	public TrainingEventID: number = 0;
	public Name: string = "";
	public NameInLocalLang: string = "";
	public CountryID: number = 0;
	public TrainingUnitID?: number;
	public TrainingUnitAcronym: string = "";
	public TrainingUnit: string = "";
	public OrganizerAppUserID?: number;
	public OrganizerFullName: string = "";
	public ParticipantCount: number = 0;
	public TrainingEventType: string = "";
	public TrainingEventStatusID?: number;
	public TrainingEventStatus: string = "";
	public EventStartDate?: Date;
	public EventEndDate?: Date;
	public Locations?: SearchTrainingEventLocations_Item[];
	public KeyActivities?: SearchTrainingEventKeyActivities_Item[];
  
}

File: GTTS/INL.SearchService/Models/SearchTrainingEvents_Item.cs
using System;
using System.Collections.Generic;

namespace INL.SearchService.Models
{
    public class SearchTrainingEvents_Item : ISearchTrainingEvents_Item
    {
        public long TrainingEventID { get; set; }
        public string Name { get; set; }
        public string NameInLocalLang { get; set; }
        public int CountryID { get; set; }
        public long? TrainingUnitID { get; set; }
        public string TrainingUnitAcronym { get; set; }
        public string TrainingUnit { get; set; }
        public int? OrganizerAppUserID { get; set; }
        public string OrganizerFullName { get; set; }
        public int ParticipantCount { get; set; }
        public string TrainingEventType { get; set; }
        public int? TrainingEventStatusID { get; set; }
        public string TrainingEventStatus { get; set; }
        public DateTime? EventStartDate { get; set; }
        public DateTime? EventEndDate { get; set; }
        public List<SearchTrainingEventLocations_Item> Locations { get; set; }
        public List<SearchTrainingEventKeyActivities_Item> KeyActivities { get; set; }
    }
}
